packages/requisitions/src/SupplierRequisition/ListView/ListView.tsx

```diff
@@ -63,10 +70,40 @@ export const SupplierRequisitionListView: FC = () => {
     [sortBy]
   );
 
+  const modalController = useToggle(false);
+
   return (
     <>
+      <NameSearchModal
+        type="supplier"
+        open={modalController.isOn}
+        onClose={modalController.toggleOff}
+        onChange={async name => {
+          modalController.toggleOff();
+
+          const createRequisition = async () => {
+            const requisition = {
+              id: generateUUID(),
+              otherPartyId: name?.id,
+            };
+
+            try {
+              const result = await onCreate(requisition);
+              invalidate();
+              navigate(result);
+            } catch (e) {
+              const errorSnack = error(
+                'Failed to create invoice! ' + (e as Error).message
```

Comment:
a little nit picky - I wouldn't expect the requisition page to be creating an invoice 😜

Reply:
Done! Though I just merged and realised I shouldve done a translation not fix the string.. oops!